plugins/modules/idrac_session.py

```diff
@@ -0,0 +1,429 @@
+                    "Severity": "OK"
+                },
+                {
+                    "Message": "A new resource is successfully created.",
+                    "MessageArgs": [],
+                    "MessageId": "IDRAC.2.9.SYS414",
+                    "RelatedProperties": [],
+                    "Resolution": "No response action is required.",
+                    "Severity": "Informational"
+                }
+            ],
+            "ClientOriginIPAddress": "100.96.37.58",
+            "CreatedTime": "2024-04-05T01:14:01-05:00",
+            "Description": "User Session",
+            "Id": "74",
+            "Name": "User Session",
+            "Password": null,
+            "SessionType": "Redfish",
+            "UserName": "root"
+        }
+error_info:
+  description: Details of the HTTP Error.
+  returned: On HTTP error
+  type: dict
+  sample: {
+        "error": {
+            "@Message.ExtendedInfo": [
+                {
+                    "Message": "Unable to complete the operation because an invalid username and/or password is entered, and therefore authentication failed.",
+                    "MessageArgs": [],
+                    "MessageId": "IDRAC.2.9.SYS415",
+                    "RelatedProperties": [],
+                    "Resolution": "Enter valid user name and password and retry the operation.",
+                    "Severity": "Warning"
+                }
+            ],
+            "code": "Base.1.12.GeneralError",
+            "message": "A general error has occurred. See ExtendedInfo for more information"
+        }
+    }
+'''
+
+
+import json
+from urllib.error import HTTPError, URLError
+from ansible_collections.dellemc.openmanage.plugins.module_utils.session_utils import SessionAPI
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import ConnectionError, SSLValidationError
+from ansible.module_utils.common.parameters import env_fallback
+from ansible_collections.dellemc.openmanage.plugins.module_utils.utils import (
+    get_dynamic_uri, remove_key)
+
+REDFISH = "/redfish/v1"
+SESSIONS = "Sessions"
+ODATA = "@odata.id"
+ODATA_REGEX = "(.*?)@odata"
+
+CREATE_SUCCESS_MSG = "The session has been created successfully."
+CREATE_FAILURE_MSG = "Unable to create a session."
+DELETE_SUCCESS_MSG = "The session has been deleted successfully."
+DELETE_FAILURE_MSG = "Unable to delete a session."
+CHANGES_FOUND_MSG = "Changes found to be applied."
+NO_CHANGES_FOUND_MSG = "No changes found to be applied."
+
+
+class Session():
+    """
+    Parent class for all session operations.
+    """
+    def __init__(self, idrac, module):
+        """
+        Initializes the object with the given idrac and module parameters.
+
+        Args:
+            idrac (object): The idrac object.
+            module (object): The module object.
+
+        Returns:
+            None
+        """
+        self.idrac = idrac
+        self.module = module
+
+    def execute(self):
```

Comment:
There is no functionality defined. If not needed, lets remove.

Reply:
Done